R/spq_perform.R

```diff
@@ -105,16 +92,43 @@ replace_prefix = function(prefix, results, .query) {
      .query[["prefixes_provided"]]
     )
 
-   dplyr::mutate(
-     results,
-     dplyr::across(
-       dplyr::where(is.character),
-       \(x) str_replace(
-         x,
-         pattern = prefixes[["url"]][prefixes[["name"]] == prefix],
-         replacement = sprintf("%s:", prefix))
-     )
+   pattern = str_replace(
+     prefixes[["url"]][prefixes[["name"]] == prefix],
+     "http:", "https?:"
```

Comment:
@lvaudor c'est pour que ça marche avec https et http, is that ok?

Reply:
oui parfait, c'est bien que ça fonctionne avec les deux